x-pack/plugins/transform/public/app/common/request.ts

```diff
@@ -8,7 +8,7 @@
 import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 
 import { HttpFetchError } from '../../../../../../src/core/public';
-import type { IndexPattern } from '../../../../../../src/plugins/data/public';
+import { DataView } from '../../../../../../src/plugins/data_views/public';
```

Comment:
Can we keep the `type` part here?

Reply:
Done in https://github.com/elastic/kibana/pull/128247/commits/b846d838066b7befa01f487a1d1caa88bcfeffbd